tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -144,3 +146,14 @@ def tap_import_certs(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         Wait(self.marionette).until(expected.element_displayed(*self._select_certs_screen_locator))
+    
+    def tap_active_wifi(self):
+        element = self.marionette.find_element(*self._active_wifi_locator)
+        element.tap()
+        Wait(self.marionette).until(expected.element_present(*self._forget_locator))
+
+    def tap_forget_wifi(self):
+        element_to_forget = Wait(self.marionette).until(expected.element_present(*self._forget_locator))
+        Wait(self.marionette).until(expected.element_displayed(element_to_forget))
+        element_to_forget.tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._screen_locator))
```

Comment:
The element with this locator, *self._screen_locator, is already displayed before the element_to_forget.tap() call, so it's a false positive.
I guess this would be sufficient here:
Wait(self.marionette).until(expected.element_not_displayed(element_to_forget))